arionBackend/transformation/esper/esper_transformer.py

```diff
@@ -14,7 +14,29 @@ class EsperTransformer(Transformer):
 	This class extends the abstract base class Transformer and implements a transformer for Esper EPL.
 	"""
 
-	def transform(self, name, queries):
+	@staticmethod
+	def generate_hierarchy(hierarchy):
+		"""
+		This methods generates the hierarchy represented as a directed graph and saves it in a given hierarchy instance.
+		:param hierarchy: the hierarchy to generate the representation for.
+		"""
+		

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1131)